frontend/src/views/RunView.vue

```diff
@@ -0,0 +1,48 @@
+<script setup>
+import { onMounted, ref } from "vue";
+
+import QcView from "@/components/QcView.vue";
+import LangQc from "@/utils/langqc.js";
+
+let serviceClient = null;
+let runWell = ref(null);
+let runCollection = null;
+let host = "https://dev-langqc.dnapipelines.sanger.ac.uk";
```

Comment:
Could this be set at runtime / figured out by the browser (frontend being served from same spot as backend)?
Otherwise, being able to set this in a config file and have the app bootstrap itself from that would be great.
(Actually the TODO in the comment below)

Reply:
It seems I can ask `window.location.origin` to divine the path, but this is very context dependent and needs preserving the moment the page loads and will foil any hopes of testing a dev or production instance without a local API. Looking into having that as a fallback but being able to inject something at build time.